subprojects/gradle/test-project/src/test/kotlin/com/avito/test/gradle/TestProjectGeneratorTest.kt

```diff
@@ -109,11 +110,11 @@ class TestProjectGeneratorTest {
     fun `generating test project with all module types is successful`() {
         val androidApp = AndroidAppModule(
             "app",
-            dependencies = """
-            implementation project(':parent:empty:library')
-            implementation project(':parent:empty:kotlin')
-            implementation project(':parent:empty:platform')
-        """.trimIndent()
+            dependencies = setOf(
+                project(projectPath = ":parent:empty:library"),
+                project(projectPath = ":parent:empty:kotlin"),
+                project(projectPath = ":parent:empty:platform")
```

Comment:
How about just `path`?

```
project(path = ":parent:empty:library"),
project(path = ":parent:empty:kotlin"),
project(path = ":parent:empty:platform")
```

I don't see reasons to duplication this knowledge.